server/graphql/v2/mutation/OrderMutations.js

```diff
@@ -494,15 +495,28 @@ const orderMutations = {
         include: [
           { association: 'paymentMethod' },
           { association: 'fromCollective', attributes: ['id', 'slug'] },
-          { association: 'collective', attributes: ['id', 'slug'] },
+          { association: 'collective', attributes: ['id', 'slug', 'HostCollectiveId'] },
         ],
       });
 
       // -- Some sanity checks to prevent issues --
       const paymentMethodIds = uniq(orders.map(order => order.PaymentMethodId).filter(Boolean));
       const ordersIds = orders.map(order => order.id);
+      let isAddedFunds;
+      let paymentMethod;
+
       for (const order of orders) {
-        const isUpdatingPaymentMethod = Boolean(fromAccount);
+        if (
+          fromAccount.HostCollectiveId !== order.collective.HostCollectiveId &&
+          fromAccount.type !== CollectiveType.USER
+        ) {
+          throw new ValidationFailed(`Currently we cannot move Added Funds authoring account to a different host`);
```

Comment:
:ice_cream: Suggested simplification

```suggestion
          throw new ValidationFailed(`Added Funds cannot be moved to a different host`);
```

Reply:
Done. 👍🏽